edweiss-app/components/lectures/slides/StudentQuestion.tsx

```diff
@@ -82,11 +82,14 @@ const StudentQuestion: ReactComponent<{ courseName: string, lectureId: string, q
 
         return (
             <>
-                <TView key={index} mb={'sm'} backgroundColor={isUser ? 'overlay0' : 'crust'} borderColor='surface0' radius={'lg'} flex={1} flexDirection='column' ml='md' style={{ right: isUser ? 0 : 10 }}>
-                    <TView borderColor={isUser ? 'crust' : 'overlay0'} bb={1} flexDirection='row' flexColumnGap={10} alignItems='center' radius={'lg'} mb={'sm'}>
-                        <Avatar size={40} name={anonym ? undefined : username} uid={anonym ? undefined : userID} />
-                        <TView flex={1}>
-                            <TText ml={16} mb={4} size={'sm'} pl={2} pt={'sm'}>{anonym ? "Anonymous" : username}, {Time.agoTimestamp(postedTime)}</TText>
+                {!answered && <TView key={index} mb={'sm'} backgroundColor='crust' borderColor='surface0' radius={'lg'} flex={1} flexDirection='column' mr={isUser ? 'md' : 'lg'} ml={isUser ? 'lg' : 'md'} style={{ right: isUser ? 0 : 10 }}>
```

Comment:
That's really good this fine touch of check that it's not yet answered.
I think in the as the students will follow it on there smartphone, displaying only unanswered question as you did is totally relevant. It's a small change that enhance the UX.

Reply:
Yes as I saw what you implemented it was an improvement that quickly came to mind

Reply:
Nice move! Happy to see that you found my feature meaningfull.